treeherder/perf/models.py

```diff
@@ -145,6 +162,25 @@ class PerformanceAlert(models.Model):
     t_value = models.FloatField(
         help_text="t value out of analysis indicating confidence "
         "that change is 'real'")
+    bug_number = models.PositiveIntegerField(null=True)
+
+    def save(self, *args, **kwargs):
+        # just a bit of extra business logic to make sure
+        # the status corresponds with other metadata
+        if self.revised_summary is not None:
+            self.status = PerformanceAlert.DUPLICATE
+        elif self.status == PerformanceAlert.DUPLICATE:
+            self.status == PerformanceAlert.UNTRIAGED
```

Comment:
Just a drive by comment.. :) Does untriaged imply it was triaged at one time, and has been 'reverted'? Or, has never been triaged and only goes through this condition once?

I was just wondering if the latter was true, if "not-triaged", might have clearer meaning.

Maybe untriaged is a widely adopted Bz term though.